builtin/providers/aws/tagsRedshift_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestDiffRedshiftTags(t *testing.T) {
+	cases := []struct {
+		Old, New       map[string]interface{}
+		Create, Remove map[string]string
+	}{
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"bar": "baz",
+			},
+			Create: map[string]string{
+				"bar": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
+		},
+		{
+			Old: map[string]interface{}{
+				"foo": "bar",
+			},
+			New: map[string]interface{}{
+				"foo": "baz",
+			},
+			Create: map[string]string{
+				"foo": "baz",
+			},
+			Remove: map[string]string{
+				"foo": "bar",
+			},
```

Comment:
Good catch, that one should be fixed too then. What's the point of having two test cases both testing the exact same thing? It's as if I asked subsequently these two questions:

`is 1+1 = 2?`
`is 1+1 still 2?`

these cases will always fail or pass both at the same time.

Do you want me to open a separate PR for the `tagsRDS_test.go`?